### PR TITLE
Add handler as object to serve static files/folders

### PR DESCRIPTION
**Pull request by jaumard**
Fix https://github.com/trailsjs/trailpack-router/issues/51

### CODE REVIEW

**Comment by jaumard** [comment]
@smythey21 can you test and confirm that now it works for you ?

File: lib/util.js [hunk 1]
```diff
@@ -11,7 +11,9 @@ module.exports = {
    */
   buildRoute (app, rawRoute) {
     const route = lib.Util.decorateRouteWithPrerequisites(app, rawRoute)
-    route.handler = _.get(app.controllers, rawRoute.handler)
+    if (typeof route.handler === 'string'){
```
Comment (konstantinzolotarev) [inline lib/util.js]: If we are using `lodash

Comment (jaumard) [inline lib/util.js]: Right :) always forgot about it, I'll correct it (the 2 call)

Comment (jaumard) [inline lib/util.js]: Done :)

**Comment by konstantinzolotarev** [comment]
@jaumard could you fix conflicts

**Comment by jaumard** [comment]
done @konstantinzolotarev  :)

**Pull request merged**